### DISCUSSION TITLE
How to remap auto-completion?

### DISCUSSION THREAD

**Discussion by mamachanko** [LunarVim/LunarVim#3732] [Q&A]
I would like to remap _auto-completion_ to something other than `<C-Space>`.

It's a little embarrassing since my past self had this figured out but forgot to commit my `~/.configl/vim/config.lua` before resetting it. 🫢

Can you help me?

**Comment by LostInTheLogs** [comment]
```lua
local cmp_mapping = require("cmp.config.mapping")
lvim.builtin.cmp.mapping["<C-Space>"] = cmp_mapping.complete()
```
